### PR TITLE
chore: Check case last event to avoid concurrent data modification by e2e tests

### PR DESCRIPTION
**Pull request by tomaszpowroznik**

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/testingsupport/controllers/TestingSupportController.java [hunk 1]
```diff
@@ -84,9 +87,18 @@ public void populateCase(@PathVariable("caseId") Long caseId, @RequestBody Map<S
         }
     }
 
+    @GetMapping("/testing-support/case/{caseId}/lastEvent")
+    public AuditEvent getLastEvent(@PathVariable("caseId") String caseId) {
+        return coreCaseDataApiV2.getAuditEvents(requestData.authorisation(), authToken.generate(), true, caseId)
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/testingsupport/controllers/TestingSupportController.java:92]: Is there a reason we are saying the request is experimental here? In #2537 we saying the request isn't experimental

**Review by lewisbirks — COMMENTED** [review]
Looks good, I completely forgot that we could do this now that we know about the case audit api. Do we want to extend this functionality to some other tests as well? The create order ones in the slow hmcts admin test all have reties due to send letter stuff as well as, the sdo and draft order stuff.

**Comment by tomaszpowroznik** [comment]
> Looks good, I completely forgot that we could do this now that we know about the case audit api. Do we want to extend this functionality to some other tests as well? The create order ones in the slow hmcts admin test all have reties due to send letter stuff as well as, the sdo and draft order stuff.

Lets trial this approach with Hearing administration scenarios, then we can update rest of test

File: service/src/main/java/uk/gov/hmcts/reform/fpl/testingsupport/controllers/TestingSupportController.java (hunk 1 above)
Comment (tomaszpowroznik) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/testingsupport/controllers/TestingSupportController.java]: changed to false to match other invocation. But from ccd pov value does not matter, it is just required marker header

**Pull request merged**